tests/query_planner/test_query_planner.py

```diff
@@ -412,7 +421,8 @@ async def test_unbound_unconnected_node(caplog):
         n2(( category biolink:PhenotypicFeature ))
         """
     )
-    qg = await expand_qg(qg, logging.getLogger())
+    await fill_categories_predicates(qg, logging.getLogger())
```

Comment:
We seem to do these together a lot. Should we make a wrapper for these two functions?